### PR TITLE
VPC Block template CVE fix May 2023

### PR DESCRIPTION
**Pull request by prankulmahajan**

### CODE REVIEW

**Comment by prankulmahajan** [comment]
```
root@prankul-vsi2:~# ibmcloud cr va icr.io/obs/storage/vpc-node-label-updater:v4.2.10
Checking security issues for 'icr.io/obs/storage/vpc-node-label-updater:v4.2.10'...

Image 'icr.io/obs/storage/vpc-node-label-updater:v4.2.10' was last scanned on Tue May  2 10:15:38 UTC 2023
The scan results show that NO ISSUES were found for the image.

OK


root@prankul-vsi2:~# ibmcloud cr va icr.io/ibm/ibm-vpc-block-csi-driver:v5.0.11
Checking security issues for 'icr.io/ibm/ibm-vpc-block-csi-driver:v5.0.11'...

Image 'icr.io/ibm/ibm-vpc-block-csi-driver:v5.0.11' was last scanned on Tue May  2 10:47:12 UTC 2023
The scan results show that NO ISSUES were found for the image.

OK
```

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 1]
```diff
@@ -3,22 +3,22 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
+                "revision": "10",
+                "release-date": "04 May 2023",
+                "cves": "CVE-2023-0361",
```
Comment (mssachan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: I think you need to add additional CVEs `CVE-2023-24536,CVE-2023-24537,CVE-2023-24538` for golang vulnerability in this list.

Comment (prankulmahajan) [reply config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: Updated

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**